src/types.ts

```diff
@@ -13,13 +13,23 @@ export interface CaptureResult {
 }
 export type CaptureCallback = (response: any, data: any) => void
 
+export type UsefulElements = 'a' | 'button' | 'form' | 'input' | 'select' | 'textarea' | 'label'
+export type AutocaptureEvents = 'click' | 'change' | 'submit'
+
+export interface AutocaptureConfig {
+    url_allowlist?: string[] | RegExp[]
+    event_allowlist?: AutocaptureEvents[]
```

Comment:
Maybe `dom_event_allowlist` do make it clear this is distinct from PostHog events (such as `$autocapture` itself)?